tools/wasme/cli/pkg/cmd/deploy/options.go

```diff
@@ -82,18 +82,20 @@ func (opts *glooOpts) addToFlags(flags *pflag.FlagSet) {
 }
 
 type istioOpts struct {
-	workload           istio.Workload
-	patchContext       string
-	istioNamespace     string
-	cacheTimeout       time.Duration
-	ignoreVersionCheck bool
+	workload             istio.Workload
+	patchContext         string
+	istioNamespace       string
+	istiodDeploymentName string
+	cacheTimeout         time.Duration
+	ignoreVersionCheck   bool
 
 	puller pull.ImagePuller // set by load
 }
 
 func (opts *istioOpts) addToFlags(flags *pflag.FlagSet) {
 	flags.StringToStringVarP(&opts.workload.Labels, "labels", "l", nil, "labels of the deployment or daemonset into which to inject the filter. if not set, will apply to all workloads in the target namespace")
 	flags.StringVarP(&opts.workload.Namespace, "namespace", "n", "default", "namespace of the workload(s) to inject the filter.")
+	flags.StringVarP(&opts.istiodDeploymentName, "istiod-name", "istiod", "istiod", "deployment name of the istiod")
```

Comment:
The `StringVarP` variant is for when we have a 1-letter shortcut for the flag (eg "namespace" / "n"). Since there's no shorthand we can just use the regular `StringVar` here.
```suggestion
	flags.StringVar(&opts.istiodDeploymentName, "istiod-name", "istiod", "deployment name of the istiod")
```
(This is the reason the tests are currently failing, it causes a runtime error).